test/framework/run.py

```diff
@@ -508,6 +509,59 @@ def test_run_cmd_stream(self):
         ])
         self.assertEqual(stdout, expected)
 
+    def test_check_log_for_errors(self):
+        fd, logfile = tempfile.mkstemp(suffix='.log', prefix='eb-test-')
+        os.close(fd)
+
+        self.assertErrorRegex(EasyBuildError, "Invalid input:", check_log_for_errors, "", [42])
+        self.assertErrorRegex(EasyBuildError, "Invalid input:", check_log_for_errors, "", [(42, IGNORE)])
+        self.assertErrorRegex(EasyBuildError, "Invalid input:", check_log_for_errors, "", [("42", "invalid-mode")])
+        self.assertErrorRegex(EasyBuildError, "Invalid input:", check_log_for_errors, "", [("42", IGNORE, "")])
+
+        input_text = "\n".join([
+            "OK",
+            "error found",
+            "test failed",
+            "msg: allowed-test failed",
+            "enabling -Werror",
+            "the process crashed with 0"
```

Comment:
Typical genuine error messages:

* `error: ...` (compiler errors)
* `Error 1`, `Error 2`, ... (`make` errors)

Reply:
Is this important for a (artificial) test? Only thing is that it needs to match the regexps used below

Reply:
not really I guess...